### ISSUE TITLE
Segmentation fault during remote-modify when insufficient permissions

### ISSUE THREAD

**Issue by sgnn7** [flatpak/flatpak#590]
Version: 0.5.2
Issue: `remote-modify` segfaults when trying to mod the system remote as a regular user

`dummy_repo` is in both system and user remotes:
```
$ # Default is system but I didn't use sudo
$ flatpak remote-modify --gpg-import=<some_key> dummy-repo

(flatpak remote-modify:25614): flatpak-WARNING **: Can't find org.freedesktop.Flatpak.SystemHelper: Error calling StartServiceByName for org.freedesktop.Flatpak.SystemHelper: GDBus.Error:org.freedesktop.DBus.Error.TimedOut: Failed to activate service 'org.freedesktop.Flatpak.SystemHelper': timed out

Segmentation fault (core dumped)

$ flatpak remote-modify --user --gpg-import=<some_key> dummy_repo
$ # Works
$ sudo flatpak remote-modify  --gpg-import=<some_key> dummy_repo
$ # Works
```

**Comment by mwleeds** [comment]
I'm not able to reproduce this using flatpak 0.9.1. Fedora just prompts me for my password if I don't use sudo. What OS are you using? Also note that you used "dummy-repo" not "dummy_repo" in the command that segfaulted but I wasn't able to reproduce it with a similar typo.

**Comment by sgnn7** [comment]
@mwleeds 'dummy-repo` was just a manual replace of the repo name since it's irrelevant but it's the smae one that I used for the whole process. OS is Ubuntu 16.04 with Flatpak PPA. It's possible that this got fixed in versions between 0.5.2 and 0.9.1 but that version was not on there at the time. I might try to do this later today to see if it got fixed.

**Comment by sgnn7** [comment]
@mwleeds This seems to not be an issue with version 0.9.1. Closing the issue.

```
$ flatpak --version                                                                                                                                       
Flatpak 0.9.1
$ sudo flatpak remote-add --no-gpg-verify testing <uri>
$ flatpak remote-modify --gpg-import=<path> <uri>                                                            
error: No remote <uri>
```
